### ISSUE TITLE
Bug blocking when received

### ISSUE THREAD

**Issue by Urgau** [SilenceIM/Silence#570]
### Bug description
The app crash every time I launch it.

### How to reproduce
Update to 0.16.3-unstable send encrypted message and disable then enable the encrypted conversation and downgrade to 0.15.7 and It's crash.

### Device info
- Device: Wiko Iggy
- Android version: 5.1
- Silence version and build ID: 0.15.7

### Link to debug log
https://gist.github.com/anonymous/2ab8a054dceb302ecd06f4d374f5b230

**Comment by ArchangeGabriel** [comment]
You cannot downgrade. Storage format in 0.16 is not retro-compatible.

**Comment by Urgau** [comment]
Yes I see this but I have no choose because in 0.16.3-unstable many of my SMS is not send.

**Comment by ArchangeGabriel** [comment]
Then you need to restore Silence with a pre 0.16 backup.
